src/amo/pages/Home/styles.scss

```diff
@@ -82,11 +82,12 @@
 .Home-SubjectShelf-list-item {
   flex-grow: 1;
   list-style-type: none;
-  margin: 15px 0;
+  margin: 0;
```

Comment:
can you just leave out margin if it's 0 here?

Reply:
Ah, yes I can.